src/components/events/EventCard/index.tsx

```diff
@@ -74,7 +74,10 @@ const EventCard = ({
       <Link
         href={`${config.eventsRoute}/${uuid}`}
         data-community={community}
-        className={`${styles.container} ${borderless ? '' : styles.bordered} ${className || ''}`}
+        data-disabled={borderless ? 'true' : 'false'}
+        className={`${styles.container} ${borderless ? 'disabled' : styles.bordered} ${
+          className || ''
+        }`}
```

Comment:
does just `data-disabled={borderless}` work?
also seeking to learn: why `'disabled'` over `''` like for className?

Reply:
`''` works for class name, 

'data-disabled={borderless}' seems to work!